packages/backend/migrations/20210422194130_create_assets_table.js

```diff
@@ -3,7 +3,7 @@ exports.up = function (knex) {
     table.uuid('id').notNullable().primary()
 
     // TigerBeetle account 2 byte ledger field representing account's asset
-    table.specificType('ledger', 'smallserial').notNullable()
+    table.specificType('ledger', 'smallserial').notNullable().unique()
```

Comment:
Need to explicitly set it as `unique` when not a primary key, even though it's unique in its nature